src/LibraryManager.Vsix/Contracts/PerProjectLogger.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.Web.LibraryManager.Contracts;
+
+namespace Microsoft.Web.LibraryManager.Vsix.Contracts
+{
+    internal class PerProjectLogger : ILogger
```

Comment:
As commented on `Logger`, it feels like this ought to be extending the `Logger` class (before it was static).  `HostInteraction` would get an instance of this type through DI.

Reply:
See my comments above. We can chat more about it in person.